src/content/en/updates/2018/10/devtools.md

```diff
@@ -0,0 +1,277 @@
+
+When a [Live Expression][LE] evaluates to a DOM node, hover over the Live Expression result to
+highlight that node in the viewport.
+
+[LE]: /web/updates/2018/08/devtools#watch
+
+<figure>
+  <img src="/web/updates/images/2018/10/hover1.png"
+       alt="Hovering over a Live Expression result to highlight the node in the viewport."/>
+  <figcaption>
+    <b>Figure 1</b>. Hovering over a Live Expression result to highlight the node in the viewport
+  </figcaption>
+</figure>
+
+## Store DOM nodes as global variables {: #store }
+
+To store a DOM node as a global variable, run an expression in the Console that evaluates to
+a node, right-click the result, and then select **Store as global variable**.
+
+<figure>
+  <img src="/web/updates/images/2018/10/store1.png"
+       alt="Store as global variable in the Console."/>
+  <figcaption>
+    <b>Figure 2</b>. Store as global variable in the Console
+  </figcaption>
+</figure>
+
+Or, right-click the node in the **DOM Tree** and select **Store as global variable**.
+
+<figure>
+  <img src="/web/updates/images/2018/10/store2.png"
+       alt="Store as global variable in the DOM Tree."/>
+  <figcaption>
+    <b>Figure 3</b>. Store as global variable in the DOM Tree
+  </figcaption>
+</figure>
+
+## Emulate page focus {: #focus }
+
+Suppose you've got a collection of buttons on a page. While the page is in focus, the currently-selected
+button has a blue border around it.
+
+<figure>
+  <img src="/web/updates/images/2018/10/focus1.png"
+       alt="The currently-selected button has a blue border around it when the page is in focus."/>
+  <figcaption>
+    <b>Figure 4</b>. The currently-selected button has a blue border around it when the page is in focus
+  </figcaption>
+</figure>
+
+However, when you click over to DevTools to do something, such as [open the Command Menu](#command-menu),
+the page loses focus, so the button loses its blue border.
+
+<figure>
+  <img src="/web/updates/images/2018/10/focus2.png"
+       alt="The blue border is gone when DevTools is in focus."/>
+  <figcaption>
+    <b>Figure 5</b>. The blue border is gone when DevTools is in focus
+  </figcaption>
+</figure>
+
+Running the new **Emulate a focused page** command from the Command Menu enables you to pretend as if the page still
+has focus while do you stuff elsewhere.
+
+<figure>
+  <img src="/web/updates/images/2018/10/focus3.png"
+       alt="Emulate a focused page."/>
+  <figcaption>
+    <b>Figure 6</b>. Emulate a focused page
+  </figcaption>
+</figure>
+
+Now the blue border remains while you do stuff in DevTools or elsewhere.
+
+<figure>
+  <img src="/web/updates/images/2018/10/focus4.png"
+       alt="The blue border remains even though the page doesn't really have focus."/>
+  <figcaption>
+    <b>Figure 7</b>. The blue border remains even though the page doesn't really have focus
+  </figcaption>
+</figure>
+
+## Initiator and priority information now in HAR imports and exports {: #HAR }
+
```

Comment:
Can we mention "HTTP Archive (HAR)" once in the text for people who don't know the acronym?